app/build.gradle

```diff
@@ -9,7 +9,7 @@ android {
     buildToolsVersion "29.0.1"
     defaultConfig {
         applicationId "org.desarrolladorslp.teckersapp"
-        minSdkVersion 16
+        minSdkVersion 27
```

Comment:
We need to discuss this. If we make the minSdkVersion this high, most potential users won't be able to install the app...

Reply:
so which one i have to leave??

Reply:
You have to know your users... in this case, the best guess we can make is that they conform to the national statistic of android users:
https://gs.statcounter.com/android-version-market-share/mobile-tablet/mexico
so target the app to a high percentage of potential users

Reply:
so i will leave 26